test/index.js

```diff
@@ -0,0 +1,296 @@
+const nock = require('nock');
+const assert = require('assert');
+const Servicetrade = require('../index');
+
+const goodStOptions = {
+    baseUrl: 'https://api.servicetrade.com',
+    username: 'good_user',
+    password: 'good_pass',
+};
+
+const badOptions = {
+    username: 'bad_user',
+    password: 'bad_pass',
+};
+
+const testOptions = {
+    baseUrl: 'https://test.host.com',
+    username: 'test_user',
+    password: 'test_pass',
+};
+
+const jobItemId = 1234;
+
```

Comment:
The `setCookie` option needs test coverage.  That feature was the whole point of updating this lib in the first place!